CONTRIBUTING.md

```diff
@@ -81,15 +81,15 @@ $ npm run watch-pages
 
 The Markdown in `src/pages` does not contain all of the Ionic documentation's content:
 
-- Paths matching `/docs/api/*` are built from the [Ionic](https://github.com/ionic-team/ionic) source code
+- Paths matching `/docs/api/*` (aka Ionic UI components) are built from the [Ionic Framework](https://github.com/ionic-team/ionic) source code
```

Comment:
Let's drop `(aka Ionic UI components)` since this will not be true for all paths under the UI components section once we add usage pages. Agreed on adding "Framework".